frontend/packages/console-app/src/components/pdb/PodDisruptionBudgetField.tsx

```diff
@@ -24,9 +24,9 @@ export const PodDisruptionBudgetField: React.FC<PodDisruptionBudgetFieldProps> =
   const pdb = getPDBResource(pdbResources, obj);
 
   return (
-    <DetailsItem label={t('console-app~PodDisruptionBudgets')} obj={obj}>
```

Comment:
```
<DetailsItem label={t('console-app~PodDisruptionBudgets')} obj={pdb}>
```
alone doesn't fix this bug?  It seems like it should...

Reply:
I had the same thought